docs/release-notes/2023-09-12-new-checkout-settings-for-bot-attack-protection.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "New Checkout Settings for Bot Attack Protection"
+slug: "2023-09-12-new-checkout-settings-for-bot-attack-protection"
+type: "added"
+excerpt: "Expanding protective measures against potential bot attacks."
+createdAt: "2023-09-12T00:00:00.000Z"
+updatedAt: "2023-09-12T00:00:00.000Z"
+---
+
+# New Checkout Settings for Bot Attack Protection
+
+In order to enhance system security, new Checkout configuration options have been made available, expanding protective measures against potential bot attacks.
+
+## What has changed?
+
+The following fields have been added to your store's orderForm configuration and can be modified via the endpoint [Update an account's orderForm configuration](https://developers.vtex.com/docs/guides/update-an-account-orderform-configuration).
+
+- `requiresLoginToPlaceOrder`: indicates whether purchases should only be completed by customers who are already authenticated in the store (via registered email).
```

Comment:
```suggestion
- `requiresLoginToPlaceOrder`: indicates whether purchases can only be completed by customers already [authenticated in the store](https://help.vtex.com/en/tutorial/authentication-page--21CkKHLKP1o41lUpGhuRUs#login-methods).
```

Reply:
I thought the "(via registered email)" part was a bit vague, and also think it is important to notice there are different authentication methods for shoppers (e.g., OAuth). So it makes sense to link the article.